### PR TITLE
chore(deployments): small improvement in the deployments docs

### PR DESCRIPTION
**Pull request by kjaskiewiczz**

### CODE REVIEW

File: backend/services/deployments/docs/management_api_v2.yml [hunk 1]
```diff
@@ -62,13 +62,13 @@ paths:
         - name: id
           in: query
           description: |
-              Deployment identifier.
+              Deployment identifier. Can be repeated to query a set of deployments.
```
Comment (merlin-northern) [inline backend/services/deployments/docs/management_api_v2.yml]:
```suggestion
              Deployment identifier. You can provide it multiple times to query a set of deployments.
```

**Review by merlin-northern — DISMISSED** [review]
the suggestion I left is according to an old rule "no passive voice in the docs" -- I have no idea if we still adhere to it.

**Review by tranchitella — DISMISSED** [review]

File: backend/services/deployments/docs/management_api_v2.yml (hunk 1 above)
Comment (kjaskiewiczz) [reply backend/services/deployments/docs/management_api_v2.yml]: done

**Review by mzedel — APPROVED** [review]

File: backend/services/deployments/docs/management_api_v2.yml (hunk 1 above)
Comment (merlin-northern) [reply backend/services/deployments/docs/management_api_v2.yml]: 😘

**Pull request merged**